src/server/pachyderm_test.go

```diff
@@ -7745,9 +7752,39 @@ func TestListDatumDuringJob(t *testing.T) {
 		}, backoff.NewTestingBackOff())
 	})
 
+	// initially since no datum chunks have been processed, we receive 0 datums
 	dis, err := c.ListDatumAll(pipelineJobInfo.PipelineJob.ID)
 	require.NoError(t, err)
-	require.Equal(t, 1, len(dis))
+	require.Equal(t, len(dis), 0)
```

Comment:
Minor nit, but the argument order should be (t, expected, actual), rather than (t, actual, expected).